docs/src/main/sphinx/release/release-354.md

```diff
@@ -0,0 +1,56 @@
+# Release 354 (XXXXXXXX Mar 2021)
+
+## General
+
+* Preferred write partitioning is now triggered automatically when the number
+  of written partitions exceeds or is equal to
+  ``preferred-write-partitioning-min-number-of-partitions``. The number of written
+  partitions is estimated from stats, therefore CTAS or insert source tables needs to have
+  statistics collected. To forcefully enable preferred partitioning,
+  ``preferred-write-partitioning-min-number-of-partitions`` needs to be set to
+  ``0`` or ``1``. To fully disable preferred write partitioning, set
+  ``use-preferred-write-partitioning`` to ``false``.
+  ({issue}`6920`)
+* Improve performance of LEFT, RIGHT and FULL JOIN when one side of the join is known to produce a single row. ({issue}`7090`)
+  
+## Security
+
+* Fix `Oauth2` repeated authentication, after previous token expires. ({issue}`7172`)
+* Support Oauth2 authentication in Trino CLI. ({issue}`7054`)
```

Comment:
nit: backtick `OAuth2` uniformly (previous release notes don't use it)

Reply:
agree on uniformity, but i think should be without backticks, as "OAuth 2.0" 

(this is how they call it on https://oauth.net/2/)